pmoves/tests/integration/test_ui_integration.py

```
@@ -0,0 +1,677 @@
+
+            task_data = init_response.json()
+            task_id = task_data["id"]
+
+            # Check task status
+            status_response = await client.get(f"/research/tasks/{task_id}")
+            assert status_response.status_code == 200
+            task = status_response.json()
+            assert task["id"] == task_id
+
+            # Try to cancel the task (to clean up)
+            cancel_response = await client.post(f"/research/tasks/{task_id}/cancel")
```

Ruff (0.14.10)

Local variable `cancel_response` is assigned to but never used

Remove assignment to unused variable `cancel_response`

(F841)